eureka/S5_lightcurve_fitting/s5_fit.py

```diff
@@ -201,7 +226,7 @@ def fitJWST(eventlabel, s4_meta=None):
                 if 'emcee' in meta.fit_method:
                     log.writelog("Starting emcee fit.")
                     model.fitter = 'emcee'
-                    lc_model.fit(model, meta, fitter='emcee')
+                    lc_model.fit(model, meta, fitter='emcee') #need to do all other fitters below here (all but lsq)
```

Comment:
What is this comment referring to?

Reply:
That was an in progress note for myself that I forgot to remove!